sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -21,7 +21,7 @@ export interface CertificateClientOptions extends coreHttp.PipelineOptions {
   /**
    * The accepted versions of the KeyVault's service API.
    */
-  serviceVersion?: "7.0" | "7.1" | "7.2";
+  serviceVersion?: "7.0" | "7.1" | "7.2" | string;
```

Comment:
Most other services I checked have it as just a string. By adding a string here I no longer have to do weird casting to `any` everywhere or introduce another type for service versions, but I lose any sort of intelliSense so I don't think there's even a reason to keep the hardcoded "7.x" options is there?